packages/components/MIGRATING.md

```diff
@@ -34,6 +34,9 @@ This document describes the changes that you need to make to your code to migrat
         -   [Tooltip](#tooltip)
             -   [Old](#old-8)
             -   [New](#new-8)
+        -   [Slider](#slider)
+            -   [Old](#old-8)
+            -   [New](#new-8)
```

Comment:
Could you move it to be in alphabetical order and also update the incorrect anchor (`#old-8` is tooltip)

Reply:
yep!